superset/result_set.py

```diff
@@ -70,8 +70,8 @@ def stringify_values(array: np.ndarray) -> np.ndarray:
 class SupersetResultSet:
     def __init__(
         self,
-        data: List[Tuple[Any, ...]],
-        cursor_description: Tuple[Any, ...],
+        data: List[Union[List[Any], Tuple[Any, ...]]],
+        cursor_description: Union[List[Any], Tuple[Any, ...]],
```

Comment:
I think this should be a `List[Union[List[Any], Tuple[Any, ...]]]` like on data.

[("user_id", "STRING", None, None, None, None, True), ("username", "STRING", None, None, None, None, True)]

Reply:
You're right, it was missing the outer `List` completely.